CONTRACT.md

```diff
@@ -0,0 +1,98 @@
+# `cumulus-sled` Contract
+
+`cumulus-sled` is a command-line interface for preparing and outputting Cumulus Messages for Cumulus Tasks. This contract defines how libraries should call `cumulus-sled` to integrate a task into a Cumulus Workflow.
+
+Every Cumulus Task includes a business function. `cumulus-sled` and a language-specific library for interacting with `cumulus-sled` are required to integrate a business function as a Cumulus Task into a Cumulus Workflow.
+
+**NOTE: This should be updated to call `cumulus-sled.zip` once we have created that package.**
+
+All `cumulus-sled` functions below are invoked via command line:
+
+```
+python ./message/message.py loadRemoteEvent '<event_json>'
+python ./message/message.py loadNestedEvent '<event_json>' '<context_json>'
+python ./message/message.py createNextEvent '<nested_event_json>' '<event_json>' '<message_config_json>'
+```
+
+These functions should be run in the order outlined above, but the output of the `loadNestedEvent` should be fed to a "business function" and the output should be the `<nested_event_json>` sent to `createNextEvent`.
```

Comment:
Instead of having a separate `loadRemoteEvent` call, could the `loadNestedEvent` call detect if this is a remote event or not, then fetch it if required?  Is there a time when you would want to call `loadRemoteEvent` but wouldn't then pass the result to `loadNestedEvent`?